install_config/aggregate_logging.adoc

```diff
@@ -473,6 +473,62 @@ $ oc delete secret logging-fluentd logging-elasticsearch \
     logging-kibana-ops-proxy
 ----
 
+
+== Upgrading the EFK Stack From 3.1 to 3.2
```

Comment:
Can this be scoped to OSE? These docs also go into origin docs and 3.1/3.2 doesn't mean anything there.